setup.py

```diff
@@ -225,6 +225,10 @@ def package_files(prefix: str, directory: str, extension: str):
     'mlflow>=2.11.1,<3.0',
 ]
 
+extra_deps['gpu-flash2'] = [
+    'flash-attn==2.5.0',
+]
```

Comment:
why `gpu-` instead of jsut `flash2`?

Reply:
should just be `flash-attention`

Reply:
wbout `flash-attn` since its the package name

Reply:
changed to `flash-attn`

Reply:
ya sounds good